bundle.md

```diff
@@ -1,6 +1,6 @@
 # Bundle Container Format
 
-This section defines a format for encoding a container as a *bundle* - a directory organized in a certain way, and containing all the necessary data and metadata for any compliant runtime to perform all standard operations against it. See also *[Mac OS application bundle*s](http://en.wikipedia.org/wiki/Bundle_%28OS_X%29) for a similar use of the term *bundle*.
+This section defines a format for encoding a container as a *bundle* - a directory organized in a certain way, and containing all the necessary data and metadata for any compliant runtime to perform all standard operations against it. See also *[Mac OS application bundles]*(http://en.wikipedia.org/wiki/Bundle_%28OS_X%29) for a similar use of the term *bundle*.
```

Comment:
Maybe [*Mac OS application bundles*][osx-bundle]?  In a pre block:

    [*Mac OS application bundles*][osx-bundle]

    [osx-bundle]: http://en.wikipedia.org/wiki/Bundle_%28OS_X%29

Although I'd personally just drop the emphasis here (the link styling is enough to set it off from the surrounding text).
[osx-bundle]: http://en.wikipedia.org/wiki/Bundle_%28OS_X%29